Makefile

```diff
@@ -1,14 +1,19 @@
+install: package.json
+	npm install
+
+lint: install
+	npm run lint
+
 test: lint
 	npm run test
 	npm run coverage
 
-lint: node_modules
-	npm run lint
+publish: test
+	npm version $(VERSION)
+	npm publish
+	@echo 'Please do git `push --follow-tags` manually!'
 
 clean:
 	rm -rf node_modules
 
-node_modules: package.json
-	npm install
-
-.PHONY: node_modules
+.PHONY: install lint test publish clean
```

Comment:
You are right, `install` shouldn't be in `.PHONY`, however it did not work as expected I had to rename it back to `node_modules` to match the directory name.